docs/en/ingest-management/elastic-agent/elastic-agent-unprivileged-mode.asciidoc

```diff
@@ -169,7 +169,23 @@ image::images/root-integration-and-unprivileged-agents.png[Agent policy tab show
 [[unprivileged-change-mode]]
 == Changing an {agent}'s privilege mode
 
-If an agent doesn't have the right level of privilege to read a data source, you can adjust the agent's privileges by adding `elastic-agent-user` to the user group that has privileges to read the data source.
+For any installed {agent} you can change the mode that it's running in by running the `privileged` or `unprivileged` subcommand.
+
+Change mode from privileged to unprivileged:
+
+[source,shell]
+----
+sudo elastic-agent unprivileged
```

Comment:
We should tell:
- this might work only on Linux (other OSes will warn it's not supported)
- It seems also it will not go in unprivileged if the Agent is enrolled with a policy which contains Elastic Defend.

Reply:
I've added a note here and in the Command Reference:
`Note that changing to `unprivileged` mode is prevented if the agent is currently enrolled in a policy that includes an integration that requires administrative access, such as the {elastic-defend} integration.`

@blakerouse is it correct that only Linux is supported?

Reply:
Switching between privileged and unprivileged works on all OS's.

Reply:
I've removed the note about the switch being supported only on Linux.